package.json

```diff
@@ -24,6 +24,7 @@
     "d3": "5.16.0",
     "data-uri-to-buffer": "3.0.1",
     "dayjs": "1.10.3",
+    "dygraphs": "2.1.0",
```

Comment:
Do we need yet another plotting library?

Reply:
Probably not, but I suggest we rip and replace once we know that vega can do what's needed.